stages/org.osbuild.ostree.deploy

```diff
@@ -136,90 +104,44 @@ def make_fs_identifier(desc):
     raise ValueError("unknown rootfs type")
 
 
-def ostree_commit_deploy(tree, inputs, osname, remote, ref, kopts):
```

Comment:
I actually think leaving this function in place (i.e. the factored version) would be useful for when I do something like `vimdiff stages/org.osbuild.ostree.deploy stages/org.osbuild.ostree.deploy.container` in the future.

Reply:
I see, so it makes it easier to isolate and see the container specific code? I guess when you push to the actually PR in osbuild, we'll take this out?

Reply:
> I see, so it makes it easier to isolate and see the container specific code? I guess when you push to the actually PR in osbuild, we'll take this out?

I just think that these two stages are so similar that having the code be easily diff-able will actually be useful over time to see exactly where the two stages differ. I'm proposing we have the refactor in the final version of the code in the PR.